src/txn/txn_rollback_to_stable.c

```diff
@@ -91,7 +93,19 @@ __rollback_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *first
         if (upd->txnid == WT_TXN_ABORTED)
             continue;
 
-        if (rollback_timestamp < upd->durable_ts || upd->prepare_state == WT_PREPARE_INPROGRESS) {
+        /*
+         * The unstable update according to the following needs to be aborted.
+         * 1. An invisible update according to the checkpoint snapshot during recovery.
+         * 2. The update durable timestamp is greater than the stable timestamp.
+         * 3. A prepared update.
+         *
+         * In a live system, there are no concurrent transaction running along with
+         * rollback to stable to verify transaction id of an update to confirm the stability
+         * of an update. But during recovery, the transaction id of an update must be validated
+         * against the checkpoint snapshot to confirm it is stability.
```

Comment:
`it is stability` it is stable

Reply:
We should add description of the fast truncate case as an example.

Reply:
Added the scenario.